app/src/main/res/values/themes.xml

```diff
@@ -31,6 +31,7 @@
         <item name="autoCompleteTextViewStyle">@style/AutoCompleteTextViewStyle</item>
         <item name="android:statusBarColor">?colorPrimary</item>
         <item name="snackbarButtonStyle">@style/snackbarButtonStyle</item>
+        <item name="materialAlertDialogTheme">@style/DaxAlertDialog</item>
```

Comment:
With this line we setup the default `MaterialAlertDialog` theme to be `DaxAlertDialog`. The only thing missing now is to always instantiate the dialog with `val alertBuilder = MaterialAlertDialogBuilder(context)` and the theme will be properly applied.